hyperspy/drawing/utils.py

```diff
@@ -483,11 +483,11 @@ def plot_images(images,
         image.
         If any signal shape is not suitable, a ValueError will be raised.
     cmap : matplotlib colormap, list, or ``'mpl_colors'``, *optional*
-        The colormap used for the images, by default read from ``pyplot``.
-        A list of colormaps can also be provided, and the images will
-        cycle through them. Optionally, the value ``'mpl_colors'`` will
-        cause the cmap to loop through the default ``matplotlib``
-        colors (to match with the default output of the
+        The colormap used for the images, by default reuse the preferences 
+        setting ``color map signal``. A list of colormaps can also be provided, 
```

Comment:
Rewrite it to e.g.:

> The colormap used for the images. Default is to use the preferences setting ``color map signal``.

Is "color map signal" correct - i.e. with no underscores, and not "cmap_signal"?

Reply:
With "color map signal", I am referring to the name of the setting as displayed in the GUI. I have tried to reworded it slightly to make it more clear.

Reply:
Looks better!